src/EditorFeatures/Core/ExternalAccess/EditorConfig/Features/EditorConfigQuickInfo.cs

```diff
@@ -0,0 +1,39 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+#nullable disable
+
+using System.Collections.Generic;
+using Microsoft.CodeAnalysis;
+using Microsoft.CodeAnalysis.Text;
+
+namespace Microsoft.CodeAnalysis.ExternalAccess.EditorConfig.Features
+{
+    internal sealed class EditorConfigQuickInfo
+    {
+        public TextSpan Span { get; }
+
+        public IEnumerable<TaggedText> Description { get; }
```

Comment:
nit: IEnumerables as fields can be risky.  it is not clear if they're lazy or not and if random work spikes may happen each time they are used.